Database/Buffer/Backend/change.ts

```diff
@@ -30,7 +30,7 @@ export async function update(request: http.Request, context: Context): Promise<h
 						unlock
 					)
 				)) ?? error("update", "Document not found")
-			context.state.waitUntil(context.setAlarm())
+			context.state.waitUntil(context.setAlarm().then())
```

Comment:
Why add `.then()`?

Reply:
`context.state.waitUntil` expected a `Promise<boolean>` before, which is changed to `Promise<void>`.

So `.then()` is converting from boolean to void. `context.setAlarm()` is a Promise<boolean>, and to not risk changing the underlaying logic, I just change the output from it, and not the underlaying type.

It is probably possible to use `as unknown as Promise<void>` but I think the real convertion with `.then()` is better.

Reply:
Since setAlarm is only used in two places and both are inside context.state.waitUntil we can safely change the return type of the function. Please remove the `return` and change return type to Promise<void> in `context.setAlarm()`.